Traceback (most recent call last):
  File "/home/dilshod/PycharmProjects/EggProject/test_/main.py", line 63, in <module>
    client.run_until_disconnected()
  File "/home/dilshod/PycharmProjects/EggProject/venv/lib/python3.10/site-packages/telethon/client/updates.py", line 96, in run_until_disconnected
    return self.loop.run_until_complete(self._run_until_disconnected())
  File "/usr/lib/python3.10/asyncio/base_events.py", line 649, in run_until_complete
    return future.result()
  File "/home/dilshod/PycharmProjects/EggProject/venv/lib/python3.10/site-packages/telethon/client/updates.py", line 38, in _run_until_disconnected
    raise self._updates_error
  File "/home/dilshod/PycharmProjects/EggProject/venv/lib/python3.10/site-packages/telethon/client/updates.py", line 308, in _update_loop
    diff = await self(get_diff)
  File "/home/dilshod/PycharmProjects/EggProject/venv/lib/python3.10/site-packages/telethon/client/users.py", line 30, in __call__
    return await self._call(self._sender, request, ordered=ordered)
  File "/home/dilshod/PycharmProjects/EggProject/venv/lib/python3.10/site-packages/telethon/client/users.py", line 87, in _call
    result = await future
telethon.errors.rpcerrorlist.AuthKeyUnregisteredError: The key is not registered in the system (caused by GetDifferenceRequest)
